tests/unit/amo/components/TestSearchResult.js

```diff
@@ -144,6 +144,97 @@ describe(__filename, () => {
     expect(root).toHaveClassName('SearchResult--theme');
   });
 
+  it('displays the thumbnail image as the default src for static theme', () => {
+    const headerImageThumb = 'https://addons.cdn.mozilla.net/thumb/1.png';
+    const addon = createInternalAddon({
+      ...fakeAddon,
+      type: ADDON_TYPE_STATIC_THEME,
+      previews: [
+        {
+          id: 1,
+          caption: 'Image 1',
+          image_url: 'https://addons.cdn.mozilla.net/full/1.png',
+          thumbnail_url: 'https://addons.cdn.mozilla.net/thumb/1.png',
+          image_size: [400, 200],
+          thumbnail_size: [200, 100],
+        },
+        {
+          id: 2,
+          caption: 'Image 2',
+          image_url: 'https://addons.cdn.mozilla.net/thumb/1.png',
+          thumbnail_url: headerImageThumb,
+          image_size: [400, 200],
+          thumbnail_size: [200, 100],
+        },
+      ],
+    });
+    const root = render({ addon });
+    const image = root.find('.SearchResult-icon');
+
+    expect(image.prop('src')).toEqual(headerImageThumb);
+  });
+
+  it('displays srcSet values if preview has multiple options', () => {
+    const headerImageThumb = 'https://addons.cdn.mozilla.net/thumb/1.png';
+    const headerImageFull = 'https://addons.cdn.mozilla.net/full/1.png';
+    const thumbWidth = 200;
+    const fullWidth = 400;
+    const addon = createInternalAddon({
+      ...fakeAddon,
+      type: ADDON_TYPE_STATIC_THEME,
+      previews: [
```

Comment:
See the suggestion above about `fakePreview`. Also, in this test you care that you get the value from `previews[1]` and not `previews[0]`, so you should override both of those with different values, so you can be sure you're getting the expected one back.